src/extension.ts

```diff
@@ -7,372 +7,452 @@
+    async createDebugAdapterDescriptor(
+        session: vscode.DebugSession,
+        executable: vscode.DebugAdapterExecutable | undefined,
+    ): Promise<vscode.DebugAdapterDescriptor | null | undefined> {
+        if (session.configuration.hasOwnProperty('consoleLogLevel')) {
+            probeRsLogLevel = session.configuration.consoleLogLevel.toLowerCase();
+        }
+
+        // Initiate either the 'attach' or 'launch' request.
+        logToConsole('INFO: Session: ' + JSON.stringify(session, null, 2));
+
+        // When starting the debugger process, we have to wait for debuggerStatus to be set to `DebuggerStatus.running` before we continue
+        enum DebuggerStatus {
+            starting,
+            running,
+            failed,
+        }
+        var debuggerStatus: DebuggerStatus = DebuggerStatus.starting;
+
+        var debugServer = new String('127.0.0.1:50000').split(':', 2); // ... provide default server host and port for "launch" configurations, where this is NOT a mandatory config
+        if (session.configuration.hasOwnProperty('server')) {
+            debugServer = new String(session.configuration.server).split(':', 2);
+            logToConsole(
+                'INFO: Debug using existing server' +
+                    JSON.stringify(debugServer[0]) +
+                    ' on port ' +
+                    JSON.stringify(debugServer[1]),
+            );
+            debuggerStatus = DebuggerStatus.running; // If this is not true as expected, then the user will be notified later.
+        } else {
+            // Find and use the first available port and spawn a new probe-rs-debugger process
+            var portfinder = require('portfinder');
+            try {
+                var port: number = await portfinder.getPortPromise();
+                debugServer = new String('127.0.0.1:' + port).split(':', 2);
+            } catch (err: any) {
+                logToConsole('ERROR: ' + JSON.stringify(err.message, null, 2));
+                vscode.window.showErrorMessage(
+                    'Searching for available port failed with: ' +
+                        JSON.stringify(err.message, null, 2),
+                );
+                return undefined;
+            }
+            var args: string[];
+            if (session.configuration.hasOwnProperty('runtimeArgs')) {
+                args = session.configuration.runtimeArgs;
+            } else {
+                args = ['debug'];
+            }
+            args.push('--port');
+            args.push(debugServer[1]);
+
+            var options = {
+                cwd: session.configuration.cwd,
+                // eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
With the new formatting, this eslint hint no longer covers RUST_LOG and DEFMT_LOG, and results in warnings. Can you please move the hint so that the warnings go away.

Reply:
This is now obsolete, since the eslint comment is gone.